### DISCUSSION TITLE
What are govc operations requiring firewall opening to ESXi?

### DISCUSSION THREAD

**Discussion by indicum** [vmware/govmomi#3040] [Community Help] [answered]
Hi,

I'm trying to know what are the govc operations that require direct access to ESXi, i.e. having firewall opened between the host where govc is executed and ESXi and not only to vcenter. But I can't find documentation about it.
Is it documented in some place or is there a way to find it without having to try executing each govc operation?

**Comment by dougm** [comment, marked as answer]
I don't think there are docs in this, but there are only a few I think of:
- import.ovf
- export.ovf
- guest.{upload,download,run}

Are there more you've tried that require ESXi access?
